admin/aws.py

```diff
@@ -126,11 +126,23 @@ def perform_copy_s3_keys(dispatcher, intent):
     """
     s3 = boto.connect_s3()
     destination_bucket = s3.get_bucket(intent.destination_bucket)
+    source_bucket = s3.get_bucket(intent.source_bucket)
     for key in intent.keys:
-        destination_bucket.copy_key(
-            new_key_name=intent.destination_prefix + key,
-            src_bucket_name=intent.source_bucket,
-            src_key_name=intent.source_prefix + key)
+        if '/_images/' in key and key.endswith('.svg'):
+            # s3cmd isn't great at choosing Content-Type. This fixes the one
+            # symptom of that we know of.
```

Comment:
I've done some poking around, and it appears that with the latest version of `s3cmd`, if we *don't* pass `--no-mime-magic`, it does the right thing for svg.